src/hooks/api/getters/useFeatures/useFeatures.ts

```diff
@@ -1,40 +1,39 @@
 import useSWR, { mutate, SWRConfiguration } from 'swr';
-import { useState, useEffect } from 'react';
+import { useCallback } from 'react';
 import { formatApiPath } from '../../../../utils/format-path';
 import handleErrorResponses from '../httpErrorResponseHandler';
+import { IFeatureToggle } from '../../../../interfaces/featureToggle';
 
-const useFeatures = (options: SWRConfiguration = {}) => {
-    const fetcher = async () => {
-        const path = formatApiPath('api/admin/features/');
-        return fetch(path, {
-            method: 'GET',
-        })
-            .then(handleErrorResponses('Features'))
-            .then(res => res.json());
-    };
-
-    const FEATURES_CACHE_KEY = 'api/admin/features/';
-
-    const { data, error } = useSWR(FEATURES_CACHE_KEY, fetcher, {
-        ...options,
-    });
+const PATH = formatApiPath('api/admin/features');
 
-    const [loading, setLoading] = useState(!error && !data);
+export interface UseFeaturesOutput {
```

Comment:
```suggestion
export interface IUseFeaturesOutput {
```

We should standardize the naming of hooks return values. I think Output works well, but let's make sure we keep it consistent when we update other hooks.

Reply:
Yeah, I was wondering about that name. Will add the `I` for now, at least.